api/build.gradle

```diff
@@ -16,11 +16,9 @@ dependencies {
             libraries.jsr305,
             libraries.animalsniffer_annotations
     compile (libraries.guava) {
-        // prefer 2.3.3 from libraries instead of 2.1.3
```

Comment:
This comment has been outdated; [Guava 28.1](https://search.maven.org/artifact/com.google.guava/guava-parent/28.1-jre/pom) uses `error_prone_annotations:2.3.2`.